scripts/inapp/ios.sh

```
@@ -2,6 +2,12 @@
 
 set -ex;
 
+export VERSION=$(grep '^version:' pubspec.yaml | sed -e 's/version: //')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)